### PR TITLE
New Component: Alert

### PR DESCRIPTION
**Pull request by aliuk2012**
Ticket: https://trello.com/c/1uueTri1

### CODE REVIEW

File: src/digitalmarketplace/components/alert/template.njk [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div data-module="dm-alert" class="dm-alert dm-alert--{{ params.type }}{%- if params.classes %} {{ params.classes }}{% endif %}"{%- for key, value in params.attributes %} {{key}}="{{value}}"{% endfor %}>
+    <h{% if params.headingLevel %}{{ params.headingLevel }}{% else %}2{% endif %} class="dm-alert--title">
```
Comment (aliuk2012) [inline src/digitalmarketplace/components/alert/template.njk]: I think the `dm-alert--title` should be `dm-alert__title`. The title is an element of component not a modification of the component.

File: src/digitalmarketplace/components/alert/template.njk [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        {{ params.titleHtml|safe if params.titleHtml else params.titleText }}
+    </h{% if params.headingLevel %}{{ params.headingLevel }}{% else %}2{% endif %}>
+    {% if params.html or params.text %}
+        <div class="dm-alert--body">
```
Comment (aliuk2012) [inline src/digitalmarketplace/components/alert/template.njk]: I think the `dm-alert--body` should be `dm-alert__body`. The body is an element of the component not a modification of the component/element.

File: src/digitalmarketplace/components/alert/_alert.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        @include govuk-responsive-margin(8, "bottom");
+        border: govuk-spacing(1) solid;
+    }
+    .dm-alert--success {
+        border-color: govuk-colour("green");
+    }
+    .dm-alert--error {
+        border-color: $govuk-error-colour;
+    }
+    .dm-alert--noticees {
```
Comment (aliuk2012) [inline src/digitalmarketplace/components/alert/_alert.scss]: I think there is a typo, shouldn't this be `.dm-alert--notice`?

File: src/digitalmarketplace/components/alert/_alert.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        border-color: govuk-colour("blue");
+    }
+    .dm-alert__title {
+        margin: 0px;
```
Comment (aliuk2012) [inline src/digitalmarketplace/components/alert/_alert.scss]: we need to set the font size and font family to be consistent. To do this we need to call  `@include govuk-font(19, $weight: bold);`

Comment (agalamatis) [reply src/digitalmarketplace/components/alert/_alert.scss]: So the only difference between body and title is the font-weight, right?

File: src/digitalmarketplace/components/alert/_alert.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        border-color: govuk-colour("green");
+    }
+    .dm-alert--error {
+        border-color: $govuk-error-colour;
+    }
+    .dm-alert--noticees {
+        border-color: govuk-colour("blue");
+    }
+    .dm-alert__title {
+        @include govuk-font(24);
```
Comment (aliuk2012) [inline src/digitalmarketplace/components/alert/_alert.scss]: This should only be font 24 if there is a param.text or param.html. May I suggest creating another class called `.dm-alert__title--large` which increases the font size to 24. by default the title should be font size 19

**Pull request closed without merging**